src/Symfony/Component/Messenger/Event/SendMessageToTransportsEvent.php

```diff
@@ -24,7 +24,7 @@
  *
  * @author Ryan Weaver <ryan@symfonycasts.com>
  */
-class SendMessageToTransportsEvent
+final class SendMessageToTransportsEvent
```

Comment:
Technically this is a BC break, isn’t it?

Reply:
no, messenger is experimental in 4.3

Reply:
Ok regarding the experimental you are right. Thanks for the feedback